multisig-ism/src/interface.sw

```diff
@@ -0,0 +1,32 @@
+library interface;
+
```

Comment:
If we ever want another contract to import this for some reason we'll need to put it into `hyperlane-interfaces`. don't feel strongly, wdyt?

Reply:
leaving here for now